examples/web/nextjs-chat/src/app/api/momento/token/config.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  AllDataReadWrite,
+  ExpiresIn,
+  TopicRole, 
+  CacheRole, 
+  TokenScope, 
+  AllTopics, 
+  AllCaches,
+  TokenScopes
+} from "@gomomento/sdk";
+
+/**
+ * First, set the scope of permissions for your tokens. 
+ * 
+ * AllDataReadWrite provides read and write permissions to all of your caches:
+ *    export const tokenPermissions: TokenScope =  AllDataReadWrite;
+ * 
+ * TokenScopes provides several functions that will return the permissions you
+ * request for a given cache and topic name.
+ *    export const tokenPermissions: TokenScope = TokenScopes.topicPublishSubscribe("default-cache", AllTopics);
+ * 
+ * You may also provide a bespoke list of permissions for each cache and topic that you have:
+ *    export const tokenPermissions: TokenScope =  {
+ *      permissions: [
+ *        {
+ *          role: CacheRole.ReadWrite | CacheRole.ReadOnly, 
+ *          cache: AllCaches | "your-cache-name"
+ *        },
+ *        {
+ *          role: TopicRole.PublishSubscribe | TopicRole.SubscribeOnly, 
+ *          cache: AllCaches | "your-cache-name",
+ *          topic: AllTopics | "your-topic-name"
+ *        }
+ *      ]
+ *    };
+ * 
+ * More information here: https://docs.momentohq.com/develop/api-reference/auth-tokens#tokenscope-objects
+ */
+export const tokenPermissions: TokenScope = TokenScopes.topicPublishSubscribe("default-cache", AllTopics);
```

Comment:
Something that I missed earlier - Matt's app allows users to dynamically select the cache from a list of existing caches. Therefore if we default to a permission that only allows a specific cache, the app will break if the user chooses a different cache from the dialog.

So I think I would change this to default to AllCaches, but possibly include your current line, commented out, with a sentence that says "to restrict browsers to only accessing a specific cache, uncomment this line" or something like that.

Reply:
Sounds good! I changed the example in the large comment block to show it can be `AllCaches` or a specific cache name